src/App.js

```diff
@@ -1,11 +1,24 @@
-import react from 'react';
-// eslint-disable-next-line import/extensions
-import Calculator from './components/calculator';
+import { Switch, Route } from 'react-router-dom';
+import Calculator from './Components/Calculator';
+import Header from './Components/header';
+import Quote from './Components/quotePage';
+import Welcome from './Components/welcome';
```

Comment:
- [ ] All these imports do not work. The App fails to compile and the error below is presented:  
  ![image](https://user-images.githubusercontent.com/45002654/149083615-4018b50c-d818-4561-a272-fecb069a43bb.png)
  
  WHY?
  The directory with the name `Components` should actually be `components` with lowercase `c`.
  Please refactor accordingly.